frontend/src/components/SideNav.tsx

```diff
@@ -258,335 +258,351 @@ export class SideNav extends React.Component<SideNavInternalProps, SideNavState>
     };
 
     return (
-      <div
-        id='sideNav'
-        className={classes(
-          css.root,
-          commonCss.flexColumn,
-          commonCss.noShrink,
-          collapsed && css.collapsedRoot,
-        )}
-      >
-        <div style={{ flexGrow: 1 }}>
-          {KFP_FLAGS.DEPLOYMENT === Deployments.MARKETPLACE && (
-            <>
+      <>
```

Comment:
You can save this nesting by moving the if out

```
if (KFP_FLAGS.HIDE_SIDENAV) {
  return null
}
return <div>...
```

Reply:
@Bobgy Nice! I force pushed a change with your suggestion